.gitignore

```diff
@@ -1,2 +1,3 @@
+composer.lock
```

Comment:
The `composer.lock` is ignored and I guess we don't need this repository to cache dependencies version.

And I think the `composer.lock` is also remove from Git version control.

Reply:
Adding the composer.lock file to version control is useful if you have a lot of dependencies so you can track down which dependencies might have broken and not respecting semver. Since time-ago doesn't rely on any dependency (except dev dependencies) it's pretty much useless to include it.

Reply:
Ok. I agree with you.